### ISSUE TITLE
Update the Workshops page

### ISSUE THREAD

**Issue by MarijaIgnj** [AccessibleAI-edu/accessibleai-edu#39]

**Comment by mutharasus** [comment]
Since for the workshop page we do not have any specific content for now and only a form, can we have a form created for this as well? 

I think having an empty workshop page would look odd for now. I can link the workshop nav item directly to the form for now.

**Comment by MarijaIgnj** [comment]
@mutharasus Yes, I agree, I made a temporary structure for the Workshops page and we will add new elements once we have content: 

https://docs.google.com/document/d/1HVIc1BdmLFbiEu_6yktJIN2gS_2x2R4-oY-fIAhwgi0/edit?usp=sharing

**Comment by mutharasus** [comment]
Please create a new form in google.

**Comment by MarijaIgnj** [comment]
@mutharasus https://forms.gle/65nznFoj1SUCGtQw7

**Comment by MarijaIgnj** [comment]
@mutharasus We need to have this made on the HTML version of the website and publish the call for the first workshop.

**Comment by mutharasus** [comment]
@MarijaIgnj  can you please review the workshops page. I think the PDF you attached is too big. Would be best to have a smaller image like the one I have. I just put a placeholder image now with some information. Would be good to not have a really long image so that it can fit on one side of the screen. So this way the future workshops will be presented above this one and then we have the ability to show multiple workshops simultaneously as well.

**Comment by MarijaIgnj** [comment]
@mutharasus OK, I see what you are saying. We will definitely need to separate upcoming workshops from the past ones somehow. 

But for now, it is OK to have this small image. I just need a title above all this ~Upcoming workshops~ and button to lead to the PDF (under Register here, a button More information). Also, clicking the image and the title should lead to the PDF, not the registration form.

Also, there is text missing above Organize a workshop button.

**Comment by mutharasus** [comment]
@MarijaIgnj  I have made these modifications. One minor detail. Instead of "Upcoming Workshops" I named the title "Upcoming". It looks very odd to have a title of "Our Workshops" immediately followed by a larger heading "Upcoming Workshops". Please review the changes in Pull Request: https://github.com/AccessibleAI-edu/accessibleai-edu/pull/101 and let me know if this is good or if you would like this modified. 

Rest of the changes are completed.

**Comment by MarijaIgnj** [comment]
@mutharasus That is fine.

**Comment by MarijaIgnj** [comment]
@mutharasus I will leave this page in the In progress category, as this only the temporary solution made to promote the upcoming workshop, we will need to make it better.
